queries.sql

```diff
@@ -1,3 +1,25 @@
-/*Queries that provide answers to the questions from all projects.*/
+/*Queries that provide answers to the questions FROM all projects.*/
```

Comment:
> Take a screenshot of the results of your queries.
Remember to add the screenshot with the results of your queries to your Pull Request description.

- [ ] According to the project requirement, you were supposed to take screenshots of these queries and include them in the pull request description, currently, I can you combine them in a .png format, I'll suggest you also include the screenshot in the pull request description as required and for easy viewing. :+1: :smile: